letsencrypt-auto-source/tests/centos6_tests.sh

```diff
@@ -86,29 +68,56 @@ fi
 
 echo "PASSED: Successfully upgraded to Python3.4 using letsencrypt-auto < 0.37.0 when only Python2.6 is present."
 
-# now bootstrap from current letsencrypt-auto, that will install python3.6 from SCL
-"$LE_AUTO" --no-self-upgrade -n > /dev/null 2> /dev/null
+# As "certbot-auto" (so without implicit --non-interactive flag set), check that the script
+# refuses to install SCL Python 3.6 when run in a non interactive shell (simulated here
+# using | tee /dev/null) if --non-interactive flag is not provided.
+cp "$LE_AUTO" /tmp/certbot-auto
+version=$(/tmp/certbot-auto --version 2>/dev/null | tee /dev/null | tail -2 | head -1)
 
-# enable SCL rh-python36
-. scl_source enable rh-python36
+if ! echo "$version" | grep -q certbot; then
+  echo "Invalid certbot version: ${version}"
+  exit 1
+fi
 
-# ensure python 3.6 is installed
-python3.6 --version > /dev/null 2> /dev/null
-RESULT=$?
-if [ $RESULT -ne 0 ]; then
-  echo "ERROR: letsencrypt-auto failed to install Python3.6 using current letsencrypt-auto when only Python2.6/Python3.4 are present."
+if [ "$version" != "$prev_version" ]; then
```

Comment:
You are right. I avoided it initially because the first approach to get the relevant previous certbot-auto script was to tie it to the certbot version. After you asked me to remove any reference to a specific version of certbot. But here I think it is relevant to refer explicitly to a version in order to simplify the tests.